src/Microsoft.Framework.TestHost/Commandline/Chain.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Framework.TestHost.Commandline
+{
+    internal struct Chain<TLeft, TDown>
```

Comment:
Ignore these. I copy-pasted for the prototype. If we like the design then I'll do a PR to move these to shared-source